backend/program/downloaders/realdebrid.py

```diff
@@ -16,16 +16,19 @@
 from RTN.parser import parse
 from RTN.patterns import extract_episodes
 from utils.logger import logger
-from utils.request import get, ping, post
+from utils.request import get, ping, post, RateLimiter
 
 WANTED_FORMATS = {".mkv", ".mp4", ".avi"}
 RD_BASE_URL = "https://api.real-debrid.com/rest/1.0"
+RATE_LIMIT_WINDOW = 1
```

Comment:
I might be being picky here, but we should add a comment for the unit of measurement here

Reply:
You are right we should - but i've just removed these constants, and changed the request and ratelimiting.
See what you think?

Introduced the concept of being able to use a rate limiter to control total requests, and a second specific endpoint
Two contexts basically - both optional

Reply:
hm, would love to cache the response, but since we need to see new responses.. that idea goes out that window :/